website/src/routes/api/(schemas)/string/index.mdx

```diff
@@ -2,6 +2,116 @@
 title: string
 ---
 
+import { Property } from '~/components';
+import { Link } from '@builder.io/qwik-city';
+
 # string
 
-> The content of this page is not yet ready. Until then just use the [source code](https://github.com/fabian-hiller/valibot/blob/main/library/src/schemas/string/string.ts).
+Schema that parses the input to string.
+
+```ts
+const schema = string(errorMessage, pipe);
+```
+
+## Parameter
+
+- `errorMessage` <Property type='string'/> = 'Invalid type'
```

Comment:
`<Property />` should support default values via props. Also `message` can also be a function. As with Modular Forms, we should probably include types in the API reference and link them. In this case it would be the [`ErrorMessage`](ot/blob/v0.19.0/library/src/types.ts#L121) type.

Reply:
Remember that `message` can also be `undefined`. You can map unions with arrays in `<Property />`:

```tsx
<Property type={['string',  'undefined']} />
```

Reply:
Implemented!
https://github.com/fabian-hiller/valibot/pull/220/commits/a224502ecd201223d41674a6d2138702efea9504